inference/core/workflows/core_steps/loader.py

```diff
@@ -68,7 +68,32 @@
 from inference.core.workflows.core_steps.classical_cv.threshold.v1 import (
     ImageThresholdBlockV1,
 )
+from inference.core.workflows.core_steps.common.deserializers import (
+    deserialize_boolean_kind,
+    deserialize_bytes_kind,
+    deserialize_classification_prediction_kind,
+    deserialize_detections_kind,
+    deserialize_dictionary_kind,
+    deserialize_float_kind,
+    deserialize_float_zero_to_one_kind,
+    deserialize_image_kind,
+    deserialize_integer_kind,
+    deserialize_list_of_values_kind,
+    deserialize_numpy_array,
+    deserialize_optional_string_kind,
+    deserialize_point_kind,
+    deserialize_rgb_color_kind,
+    deserialize_string_kind,
+    deserialize_video_metadata_kind,
+    deserialize_zone_kind,
+)
 from inference.core.workflows.core_steps.common.entities import StepExecutionMode
+from inference.core.workflows.core_steps.common.serializers import (
+    serialise_image,
+    serialise_sv_detections,
```

Comment:
```suggestion
    serialize_sv_detections,
```

Reply:
existing function in `serializers` module - to avoid bc - let's live with this